tests/virt_autotest/libvirt_routed_virtual_network.pm

```diff
@@ -89,9 +89,9 @@ sub run_test {
 
         #Wait for guests attached interface from virtual routed network
         sleep 30;
-        my $net1 = (is_sle('=11-sp4') && (get_var('XEN') || check_var('SYSTEM_ROLE', 'xen') || check_var('HOST_HYPERVISOR', 'xen'))) ? 'netfront' : 'vnet_routed';
+        my $net1 = is_sle('=11-sp4') ? 'br123' : 'vnet_routed';
         test_network_interface("$guest", mac => $mac1, gate => $gate1, routed => 1, target => $target1, net => $net1);
-        my $net2 = (is_sle('=11-sp4') && (get_var('XEN') || check_var('SYSTEM_ROLE', 'xen') || check_var('HOST_HYPERVISOR', 'xen'))) ? 'netfront' : 'vnet_routed_clone';
+        my $net2 = is_sle('=11-sp4') ? 'br123' : 'vnet_routed_clone';
```

Comment:
Are you sure about this replacement that 11sp4 needs br123? IMHO, the tested networks should be the same for 11 and 12 products. Br123 seems never the test target in the 4 kinds of networks.

Reply:
Good point, 
'br123' work for save_guest_ip func on sles11sp4 vm host, to get guest ip address and save guest ip address to  /etc/hosts to let host connect guest system with guest name via ssh. 
why just only used 'br123' on sles11sp4, because of save_guest_ip func used with output from domiflist via virsh. however, just only output from domiflist via virsh on s11sp4 be different with the other sles version (i.e s12+)
For example
SLES11SP4
virsh domiflist $guest
Interface TYPE SOURCE MAC
----------------------------------------------------
vnet0  bridge br123 XX.XX.XX.XX
vnet1  bridge br0 XX.XX.XX.XX

SLES12+
virsh domiflist $guest
Interface TYPE SOURCE MAC
----------------------------------------------------
vnet0  bridge br123 XX.XX.XX.XX
vnet1  vnet_host_bridge br0 XX.XX.XX.XX

So, have to used with br123 as keyword to save guest ip address on s11sp4 vm host